modules/ossm-about-collecting-ossm-data.adoc

```diff
@@ -31,3 +31,12 @@ $ oc adm must-gather --image=registry.redhat.io/openshift-service-mesh/istio-mus
 ----
 $ oc adm must-gather --image=registry.redhat.io/openshift-service-mesh/istio-must-gather-rhel8:2.4 gather <namespace>
 ----
+
```

Comment:
If the list on lines 35 - 42 should be considered as part of step 2, add a plus sign (`+`) on line 34 to nest the list under the step.